packages/front-end/components/Auth/CreateOrJoinOrganization.tsx

```diff
@@ -250,8 +268,33 @@ export default function CreateOrganization(): ReactElement {
               GrowthBook.
             </div>
           </div>
-        )}
-      </WelcomeFrame>
-    </>
+        )}{" "}
+      </div>
+    </div>
   );
-}
+
+  if (showFrame) {
+    return (
+      <>
+        <WelcomeFrame leftside={leftside} loading={loading}>
+          <a
```

Comment:
i think we are meant to use <Link> here

Reply:
oh wait it shouldnt matter since it doesnt link anywhere

Reply:
> i think we are meant to use here

Use `<Link>`? Github I think ate your tag and doesn't display it.  At any rate, yeah since it doesn't go anywhere the linter doesn't complain about this.  If it went anywhere I think the linter would complain.

Reply:
yeah i relized after i made this comment it didnt go anywhere no need to update it :)